src/includes/configuration/auto-session-tracking/android.mdx

```diff
@@ -1,3 +1,5 @@
+To benefit from the health data, you must use at least version 2.1.0 of the Android SDK, and enable the collection of release health metrics in the AndroidManifest.xml file. Release health is captured, by default, in our Android version 3.0 and higher, unless you specifically disabled collecting it in the initialization options for the SDK.
```

Comment:
should we stick to either `in the AndroidManifest.xml file` or `in the initialization options`? because they are 2 different ways of configuring the Android SDK.

Reply:
Good point. We are discussing two version of the SDK here, 2.1.-, where you need to modify the `AndroidManifest.xml`file, or version 3.0 where it's captured by default unless disabled. 

I think, to make this more clear, we might simply focus on most recent, so remove the first sentence altogether. What do you think?

Reply:
I've pulled your suggestion below, but we can modify further, as appropriate.